docker-compose.yaml

```diff
@@ -1,48 +1,34 @@
 version: '3'
+
 services:
   ffc-demo-claim-service:
     build: .
     image: ffc-demo-claim-service
-    container_name: ffc-demo-claim-service
     depends_on:
       - ffc-demo-claim-postgres
+    env_file:
+      - variables/message-queues.env
+      - variables/postgres.env
     environment:
-      POSTGRES_DB: ffc_demo_claims
-      POSTGRES_PASSWORD: ppp
-      POSTGRES_USERNAME: postgres
       POSTGRES_HOST: ffc-demo-claim-postgres
       POSTGRES_PORT: 5432
       MESSAGE_QUEUE_HOST : host.docker.internal
       MESSAGE_QUEUE_PORT : 5672
-      CALCULATION_QUEUE_ADDRESS : calculation
-      CALCULATION_QUEUE_USER : artemis
-      CALCULATION_QUEUE_PASSWORD : artemis
-      SCHEDULE_QUEUE_ADDRESS : schedule
-      SCHEDULE_QUEUE_USER : artemis
-      SCHEDULE_QUEUE_PASSWORD : artemis
     networks:
-      - claim-postgres
-    ports:
-      - "3003:3003"
+      - ffc-demo-claim

```

Comment:
The `ffc-demo-claim` network can be removed altogether so we make use of docker-compose's default networking between containers in the same stack. This has been done for most services as part of the equivalent PRs but can be regarded as a separate issue if you prefer.

Reply:
I've removed that network now, makes sense to do it alongside the other changes.